<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArcGIS_Window</class>
 <widget class="QWidget" name="ArcGIS_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_16">
         <item row="1" column="3">
          <widget class="QSlider" name="slider_frame">
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Frame</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Camera</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QComboBox" name="mission_select">
           <item>
            <property name="text">
             <string>Grand Canyon</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hawaii</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pyrenees</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Snowdon</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLabel" name="value_frame">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="4">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>OrbitGeoElementCameraController</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Angle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slider_angle">
              <property name="maximum">
               <number>180</number>
              </property>
              <property name="value">
               <number>45</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Zoom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slider_zoom">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
              <property name="value">
               <number>500</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Speed</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slider_speed">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="camera_follow">
           <property name="text">
            <string>Follow</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="4">
          <widget class="QPushButton" name="button_play">
           <property name="text">
            <string>Play</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Mission</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Animate_Symbol" name="widget_9" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Animate_Symbol</class>
   <extends>QWidget</extends>
   <header location="global">Demo/Animate_Symbol.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
